src/client/feed/feedReducer.js

```diff
@@ -0,0 +1,12 @@
+import { UPDATE_FEED_RESULTS } from '../search/SearchActions';
+
+const feed = (state = { 'posts': [] }, action) => {
+  switch (action.type) {
+    case UPDATE_FEED_RESULTS:

```

Comment:
I think these should all be defined in Actions files, not Reducers. Definitely should be one or the other, not a mix